articles/install-guide/update.md

```diff
@@ -16,7 +16,53 @@ This article assumes that you already have the QDK installed. If you are install
 
 The update steps depend on your development environment. Choose your environment from the following sections.
 
-## Python
+## Updating Q# Projects 
+
+1. First, install the latest version of the [.NET Core SDK 3.0](https://dotnet.microsoft.com/download) and run the following command in the command prompt:
+```bash
+dotnet --version
+```
+ Verify the output is 3.0.100 or higher, then follow the instructions below for your setup.
+
+### In Visual Studio
+ 
+ 1. Run Build > Clean Solution 
+ 1. Update the target framework in each of your .csproj files to netcoreapp3.0 (or netstandard2.1 if it is a library project)
+ 1. Save and close all files in your solution
+ 1. Select Tools > Command Line > Developer Command Prompt
+ 1. For each project in the solution, run the following command:
+ ```bash
+ dotnet add Path\To\[project_name].csproj package Microsoft.Quantum.Development.Kit`
+ ```
+ Close the command prompt and select Build > Build Solution (do *not* select Rebuild Solution, as rebuilding will initially fail)
+
+### In Visual Studio Code
+
+1. In Visual Studio Code, open the folder containing the project to update
+1. Select Terminal > New Terminal
+1. Follow the instructions for updating using the command line
+
+### Using the command line
+
+1. Navigate to the folder containing your project file
+2. Run the following commands:
+```bash
+dotnet clean MyProj.csproj
+```
+
+and
+
+```bash
+dotnet add package Microsoft.Quantum.Development.Kit
+```
+3. Update the target framework in each of your .csproj files to netcoreapp3.0 (or netstandard2.1 if it is a library project)
```

Comment:
Actually, we might need to move this up between dotnet clean and dotnet add package, since the add package will I think complain otherwise, and the clean will fail after!